cluster/addons/kube-network-policies/kube-network-policies.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kube-network-policies
+  namespace: kube-system
+  labels:
+    tier: node
+    app: kube-network-policies
+    k8s-app: kube-network-policies
+    addonmanager.kubernetes.io/mode: Reconcile
+spec:
+  selector:
+    matchLabels:
+      app: kube-network-policies
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: kube-network-policies
+        k8s-app: kube-network-policies
+    spec:
+      hostNetwork: true
+      tolerations:
+      - operator: Exists
+        effect: NoSchedule
+      serviceAccountName: kube-network-policies
+      containers:
+      - name: kube-network-policies
+        image: aojea/kube-netpol:v0.1.0 # this is temporal we need to move it to kubernetes-sigs
+        args:
+        - /bin/netpol
+        - -v
+        - "4"
+        volumeMounts:
+        - name: lib-modules
+          mountPath: /lib/modules
+          readOnly: true
+        resources:
+          requests:
+            cpu: "100m"
+            memory: "50Mi"
+        securityContext:
+          privileged: true
+          capabilities:
+            add: ["NET_ADMIN"]
+      volumes:
+      - name: lib-modules
+        hostPath:
+          path: /lib/modules
```

Comment:
> You should be able to remove this. I don't think `nft` ever manually modprobes modules like `iptables` does.

and even if we're using the fallback iptables mode of kube-netpol, it won't have to load the `ip_tables` module itself, because kubelet will already have created the `KUBE-IPTABLES-HINT` chain before any pods are started. (And `xt_NFQUEUE` gets loaded indirectly via udev, not directly by the iptables binary, so we don't need it for that either.)